src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,393 @@
+      ],
+      "how": [
+        "System users are now excluded from the metric report, we didn't exclude them from the total, but they were excluded from the per-day aggregation"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/fd6465d0f95d78210401f0d0be9c16e1290341af"
+      ]
+    },
+    {
+      "what": [
+        "The timestamp in the details of an action log being different as the one in search results"
+      ],
+      "how": [
+        "The custom format function IocFullDateTime was applied to the timestamp"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/a4596fb283b42822fe3bb4c81c4cbc4f43d7bfe4"
+      ]
+    },
+    {
+      "what": [
+        "Strict record value duplicate detection, which was preventing record creation with same values for fields for strict duplicate detection validation"
+      ],
+      "how": [
+        "Validating duplication only for selected field"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/eac2f07"
+      ]
+    },
+    {
+      "what": [
+        "The issue caused by the .Load(...) workflow service function, not including sub-workflows which excluded them from the cache and made them unavailable in the workflow execution step."
+      ],
+      "how": [
+        "The patch now includes sub-workflows in the initial load thus resolving the issue."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/d64b732c25273ca3b588cc94ebb716a1a0fabca7"
+      ]
+    },
+    {
+      "what": [
+        "The userID, once filtered by it, you could not remove the filter which made the server filter by 0 which will never exist."
+      ],
+      "how": [
+        "The date-time issue, both from and to were using the same (from) timestamp which returns nothing.", 
+        "Now they are using separate timestamp."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/1fa5f39ab2132b196b543bb8b4f376abca3bc24c",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/c63df822a2a276bdc29bdf792c7527fa74e1f9de"
+      ]
+    },
+    {
+      "what": [
+        "Exporting a NS where the name used characters like /, . broke the export because they broke the URL.",
+        "Same for record export but with module name."
+      ],
+      "how": [
+        "URI encoding the filename and replacing dots with -, which caused the URL to be broken"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/1f0ea56cf1348fc60a51f489e61afe69d51ca736",
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/920cc5a0d62f47272a15090cfe1f4b35de3efca1"
+      ]
+    },
+    {
+      "what": [
+        "JSON codecs getting set to the field name instead of values, when editing a module/configuring field mapping."
```

Comment:
What problem did this cause?

Here, the `JSON` codec requires you to specify where the JSON object is located, **not** the name of the value.
Since values are stored in `values`, the value has to be `values` (where the JSON is).
The bug told Corteza to look for the column with the field name which probably never existed.